src/loader/loader.py

```diff
@@ -39,16 +42,17 @@ def fetch_packages(*, pypi_client: PyPIServices) -> list[tuple[str, str]]:
 def load_packages(packages: list[tuple[str, str]], *, http_client: Client, access_token: str) -> None:
     """Load all of the given packages into the Dragonfly API, using the given HTTP session and access token."""
     payload = [{"name": name, "version": version} for name, version in packages]
+    logger.info("Loading packages: %s", payload)
     headers = build_authorization_header(access_token)
 
-    res = http_client.post(f"{Settings.base_url}/batch/package", json=payload, headers=headers)
-    res.raise_for_status()
+    http_client.post(f"{Settings.base_url}/batch/package", json=payload, headers=headers)
 
 
 def main(*, http_client: Client, pypi_client: PyPIServices) -> None:
     """Run the loader."""
     access_token = get_access_token(http_client=http_client)
 
     packages = fetch_packages(pypi_client=pypi_client)
+    logger.info("Fetched packages: %s", packages)
```

Comment:
```suggestion
```
I don't think we should log this because we log the same thing later on when we're loading the packages. We end up with 2 x 100 packages logged which is not ideal. We could also just do something like, `fetched 100` packages maybe. doesn't really matter, up to you